services/data-info/src/data_info/routes/stats.clj

```diff
@@ -9,6 +9,8 @@
 
 (defroutes* stat-gatherer
 
+            ; FIXME Update metadactyl exception handling when data-info excptn hndlg updated
+            ; metadactyl catches exceptions thrown from this EP.
```

Comment:
@psarando Exceptions thrown in Donkey need to be updated. Also, there are some which are caught and rethrown, and I haven't spent enough time analyzing them yet. 

But there are definitely instances of exception throwing in Donkey which can be updated.